### PR TITLE
[OETHb] Deploy 013 - Fixed Rate Dripper

### PR DESCRIPTION
**Pull request by shahthepro**
## Deployment
- FixedRateDripper: [0xA3A4759DF6687Cd2573B1399b68118BB86EccDaE](<https://basescan.org/address/0xA3A4759DF6687Cd2573B1399b68118BB86EccDaE>)

## Governance
- Upgrade OETHb Dripper proxy to use the new FixedRateDripper implementation

## Deploy checklist

Two reviewers complete the following checklist:

```
- [ ] All deployed contracts are listed in the deploy PR's description
- [ ] Deployed contract's verified code (and all dependencies) match the code in master
- [ ] The transactions that interacted with the newly deployed contract match the deploy script.
- [ ] Governance proposal matches the deploy script
- [ ] Smoke tests pass after fork test execution of the governance proposal
```

### CODE REVIEW

**Comment by naddison36** [comment]
- [x] All deployed contracts are listed in the deploy PR's description
- [x] Deployed contract's verified code (and all dependencies) match the code in master
  - [x] FixedRateDripper
- [x] The transactions that interacted with the newly deployed contract match the deploy script.
- [x] Governance proposal matches the deploy script
- [ ] Smoke tests pass after fork test execution of the governance proposal
  - there are currently five `FixedRateDripper` tests failing in the Base fork tests

```
sol2uml diff 0xA3A4759DF6687Cd2573B1399b68118BB86EccDaE .,node_modules -n base
Compared the "FixedRateDripper" contract with address 0xA3A4759DF6687Cd2573B1399b68118BB86EccDaE on base
to local files under folders ".,node_modules"

match   @openzeppelin/contracts/token/ERC20/IERC20.sol
match   @openzeppelin/contracts/token/ERC20/utils/SafeERC20.sol
match   @openzeppelin/contracts/utils/Address.sol
match   @openzeppelin/contracts/utils/math/SafeMath.sol
match   contracts/governance/Governable.sol
match   contracts/harvest/Dripper.sol
match   contracts/harvest/FixedRateDripper.sol
match   contracts/interfaces/IBasicToken.sol
match   contracts/interfaces/IStrategy.sol
match   contracts/interfaces/IVault.sol
match   contracts/token/OUSD.sol
match   contracts/utils/Helpers.sol
match   contracts/utils/Initializable.sol
match   contracts/utils/InitializableERC20Detailed.sol
match   contracts/utils/StableMath.sol
match   contracts/vault/VaultStorage.sol
```


```

  1) ForkTest: OETHb FixedRateDripper
       Should collect reward at fixed rate:

      AssertionError: Expected "2914627961147493496" to be less than or equal 2914592460816018888
      + expected - actual

       {
      -  "_hex": "0x2872b666a3f86dc8"
      +  "_hex": "0x2872d6b0349e0c78"
         "_isBigNumber": true
       }
      
      at Proxy.<anonymous> (test/helpers.js:16:32)
      at Proxy.methodWrapper (node_modules/chai/lib/chai/utils/addMethod.js:57:25)
      at Context.<anonymous> (test/dripper/fixed-rate-dripper.base.fork-test.js:49:34)
      at processTicksAndRejections (node:internal/process/task_queues:95:5)
      at runNextTicks (node:internal/process/task_queues:64:3)
      at listOnTimeout (node:internal/timers:545:9)
      at processTimers (node:internal/timers:519:7)

  2) ForkTest: OETHb FixedRateDripper
       Should allow strategist/governor to change rate:
     Error: Transaction reverted without a reason string
    at <UnrecognizedContract>.<unknown> (0x02f2c609950e90934ce99e58b4d7326ad0d7f8d6)
    at <UnrecognizedContract>.<unknown> (0x02f2c609950e90934ce99e58b4d7326ad0d7f8d6)
    at processTicksAndRejections (node:internal/process/task_queues:95:5)
    at HardhatNode._mineBlockWithPendingTxs (node_modules/hardhat/src/internal/hardhat-network/provider/node.ts:1840:23)
    at HardhatNode.mineBlock (node_modules/hardhat/src/internal/hardhat-network/provider/node.ts:517:16)
    at EthModule._sendTransactionAndReturnHash (node_modules/hardhat/src/internal/hardhat-network/provider/modules/eth.ts:1532:18)
    at HardhatNetworkProvider.request (node_modules/hardhat/src/internal/hardhat-network/provider/provider.ts:123:18)
    at TracerWrapper.request (node_modules/hardhat-tracer/src/wrapper.ts:30:16)
    at EthersProviderWrapper.send (node_modules/hardhat-deploy-ethers/src/internal/ethers-provider-wrapper.ts:13:20)
  

  3) ForkTest: OETHb FixedRateDripper
       Should not allow anyone else to change rate:
     AssertionError: Expected transaction to be reverted with "Caller is not the Strategist or Governor", but other reason was found: ""
  

  4) ForkTest: OETHb FixedRateDripper
       Should allow to disable rate:
     Error: Transaction reverted without a reason string
    at <UnrecognizedContract>.<unknown> (0x02f2c609950e90934ce99e58b4d7326ad0d7f8d6)
    at <UnrecognizedContract>.<unknown> (0x02f2c609950e90934ce99e58b4d7326ad0d7f8d6)
    at processTicksAndRejections (node:internal/process/task_queues:95:5)
    at HardhatNode._mineBlockWithPendingTxs (node_modules/hardhat/src/internal/hardhat-network/provider/node.ts:1840:23)
    at HardhatNode.mineBlock (node_modules/hardhat/src/internal/hardhat-network/provider/node.ts:517:16)
    at EthModule._sendTransactionAndReturnHash (node_modules/hardhat/src/internal/hardhat-network/provider/modules/eth.ts:1532:18)
    at HardhatNetworkProvider.request (node_modules/hardhat/src/internal/hardhat-network/provider/provider.ts:123:18)
    at TracerWrapper.request (node_modules/hardhat-tracer/src/wrapper.ts:30:16)
    at EthersProviderWrapper.send (node_modules/hardhat-deploy-ethers/src/internal/ethers-provider-wrapper.ts:13:20)
```

File: contracts/deployments/base/.migrations.json [hunk 1]
```diff
@@ -11,5 +11,6 @@
   "010_upgrade_vault_core": 1724933115,
   "011_transfer_governance": 1725984158,
   "012_claim_governance": 1725984162,
-  "013_revoke_admin_role": 1727204761
+  "013_revoke_admin_role": 1727204761,
+  "013_fixed_rate_dripper": 1727409372
```
Comment (sparrowDom) [inline contracts/deployments/base/.migrations.json:15]: nit: there are 2 `013` migrations

**Comment by sparrowDom** [comment]
I've verified that: 
- [x] All deployed contracts are listed in the deploy PR's description
- [x] Deployed contract's verified code (and all dependencies) match the code in master
- [x] The transactions that interacted with the newly deployed contract match the deploy script.
- [x] Governance proposal matches the deploy script
- [ ] Smoke tests pass after fork test execution of the governance proposal

```
npx sol2uml diff 0xA3A4759DF6687Cd2573B1399b68118BB86EccDaE .,node_modules --network base
Compared the "FixedRateDripper" contract with address 0xA3A4759DF6687Cd2573B1399b68118BB86EccDaE on base
to local files under folders ".,node_modules"

match   @openzeppelin/contracts/token/ERC20/IERC20.sol
match   @openzeppelin/contracts/token/ERC20/utils/SafeERC20.sol
match   @openzeppelin/contracts/utils/Address.sol
match   @openzeppelin/contracts/utils/math/SafeMath.sol
match   contracts/governance/Governable.sol
match   contracts/harvest/Dripper.sol
match   contracts/harvest/FixedRateDripper.sol
match   contracts/interfaces/IBasicToken.sol
match   contracts/interfaces/IStrategy.sol
match   contracts/interfaces/IVault.sol
match   contracts/token/OUSD.sol
match   contracts/utils/Helpers.sol
match   contracts/utils/Initializable.sol
match   contracts/utils/InitializableERC20Detailed.sol
match   contracts/utils/StableMath.sol
match   contracts/vault/VaultStorage.sol
```

**Review by sparrowDom — APPROVED** [review]

File: contracts/deployments/base/.migrations.json (hunk 1 above)
Comment (shahthepro) [reply contracts/deployments/base/.migrations.json]: Thanks, have updated it

**Pull request merged**